samples/msgext-unfurling-ac-loop-components/csharp/AdapterWithErrorHandler.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+using Microsoft.Bot.Builder.TraceExtensions;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.BotBuilderSamples
```

Comment:
Namespace name looks wrong here.

Reply:
Please try to create any new .cs file and verify.

Reply:
Fixed